public/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <%= process.env.VUE_APP_TIER !== '' ? '<meta name="robots" content="noindex,nofollow">' : '' %><!-- an empty string in this case means the 'prod' version of the application   -->
+      <!-- Primary Meta Tags -->
+      <title>
+        <%= VUE_APP_TIER %>
+          <%= VUE_APP_TITLE %>
+      </title>
+      <meta name="title" content="U.S. Groundwater Conditions">
+      <meta name="description" content="">
+      <!-- Open Graph / Facebook -->
+      <meta property="og:type" content="website">
+      <meta property="og:url" content="https://labs.waterdata.usgs.gov/visualizations/gw-conditions/index.html#/">
+      <meta property="og:title" content="U.S. Groundwater Conditions">
+      <meta property="og:description" content="">
+      <meta property="og:image" content="https://labs.waterdata.usgs.gov/visualizations/gw-conditions/SnowToFlowMetacard.jpg">
```

Comment:
Should we make an issue or something to remember to change this?

Reply:
This is on the viz "final steps to release" checklist which I can add to the issues.

Reply:
Oh no worries if it is captured elsewhere!